Description: Bumbs the source and target version options for the java compiler.
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -130,10 +130,10 @@
 <!-- ========== Compiler Defaults ========================================= -->
 
   <!-- Version of java class files to generate. -->
-  <property name="target.version" value="1.2"/>
+  <property name="target.version" value="1.3"/>
 
   <!-- Version of java source to accept -->
-  <property name="source.version" value="1.2"/>
+  <property name="source.version" value="1.3"/>
 
   <!-- Should Java compilations set the 'debug' compiler option? -->
   <property name="compile.debug"           value="true"/>
